lib/diagnostic.js

```diff
@@ -4,8 +4,11 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-const min = function min() {
-
+const min = function min(sum, sum2) {
+  return Math.min.apply(sum, sum2);
+    }
+  }
+  return min();
```

Comment:
This also breaks everything, you are calling the function your in with no parameters and trying to return it here